intranet/static/js/eighth/statistics.js

```diff
@@ -0,0 +1,66 @@
+/* global $ */
+$(document).ready(function() {
+    if (raw_data.keys.length > 0) {
+        $("#members-chart").show();
+        var canvas = $("#members-chart")[0];
+        var data = {
+            labels: raw_data.keys,
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'raw_data' is not in camel case.](https://www.codacy.com/app/tjcsl/ion/file/2620617210/issues/source?bid=3176160&fileBranchId=3299251#l7)